richmenu-template/template-01.md

```diff
@@ -0,0 +1,46 @@
+---
+marp: true
+size: 16:9
+---
+<style>
+section {
```

Comment:
I'd like to separate the common css part. However, it was a difficult. therefore, I'll work the next issue.

Prioritize release over perfection. :bow: :bow: